packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -290,7 +297,7 @@ export const ChatMessageComponentAsRichTextEditBox = (
             e.stopPropagation();
           }}
           id={'submitIconWrapper'}
-          data-testId={strings.editBoxSubmitButton}
+          data-testId={'chat-message-rich-text-edit-box-submit-button'}
```

Comment:
Should this test ID string be pulled out into a const as it's used in multiple places?